### DISCUSSION TITLE
Custom (static) User-Agent

### DISCUSSION THREAD

**Discussion by gunar** [upptime/upptime#654] [Ideas]
Today, requests from `upptime` show up having the user agent of `axios/0.27.2`. I'd be nice if we were to customize that message so that requests from `upptime` are easily identifiable when looking at the logs. This would be especially useful in case `upptime` identifies a downtime but you have a hard time tracking down exactly what happened (did my service fail? did upptime fail? was it just a networking issue?)

**Comment by AnandChowdhary** [comment]
You can customize the user agent like so: https://upptime.js.org/docs/configuration/#user-agent

**Reply by gunar** [reply]
This did not work.
